src/content/docs/guides/usage/data/shape-data.mdx

```diff
@@ -225,3 +246,21 @@ sort -x
 ```
 
 Prepending the field with `-` reverses the sort order.
+
+## Invert event ordering with `reverse`
+
+Use `reverse` to invert the order of a stream of events:
+
+```tql
+from {x: 1},
+     {x: 2},
+     {x: 3}
+reverse
+```
+
+```tql
+{x: 3}
+{x: 2}
+{x: 1}
+```
+
```

Comment:
```suggestion
```

Reply:
I've added a TODO to the README about this.